app/components/App/components/Header/Header.less

```diff
@@ -1,15 +1,14 @@
 @import (reference) "~seek-style-guide/theme";
 
 .root {
-  height: (@grid-row-height * 14);
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
+  height: (@grid-row-height * 16);
+  padding-top: (@grid-row-height * 2);
 }
 
 .logo {
   display: flex;
   align-items: center;
+  margin-left: -(@grid-gutter-width / 2);
```

Comment:
I still instinctively twitch when I see a negative margin.  Fine to get it unbroken though!